pages/dayTwo/index.js

```diff
@@ -0,0 +1,25 @@
+import { data } from "../../components/utils/data";
+import RoomNav from "../../components/RoomNav";
+import Header from "../../components/Header";
+import StyledLink from "../../components/StyledLink";
+
+const filteredByDay = data.filter((talk) => talk.date === "24.11.2022");
+console.log(filteredByDay);
+
+function DayTwo() {
+  return (
+    <>
+      <Header>
+        <StyledLink href="/dayOne">Mittwoch 23.11.</StyledLink>
+        <StyledLink href="#" style={{ background: "darkgray" }}>
+          Donnerstag 24.11.
+        </StyledLink>
+        <StyledLink href="/dayThree">Freitag 25.11.</StyledLink>
+      </Header>
+      <RoomNav talks={filteredByDay} />
```

Comment:
From the name of the component, you could suspect that only the navigation to switch between the rooms, is actually inside that component. But you also have your talk list inside there.

You should either rename the component or better separate the room navigation from your talk list. Both can then be imported in your pages.